### ISSUE TITLE
Doesn't work, doesn't show any errors

### ISSUE THREAD

**Issue by AndrewRayCode** [adamwalzer/vscode-scss-lint#7]
```
$ gem install scss_lint
Fetching: scss_lint-0.53.0.gem (100%)
Successfully installed scss_lint-0.53.0
1 gem installed
```

Install this extension. Don't do anything else (meaning don't add configuration settings).

Restart VSCode

Open `app/style/forms.scss`, or whatever.scss, add:

    $background-color:rgba(2,ab3);

No errors

Save file

No errors

No warnings either about invalid configuration.

**Comment by donni106** [comment]
Hi, did you check this issue: https://github.com/adamwalzer/vscode-scss-lint/issues/2?

Did you try with scss_lint 0.49.0, as saying in the docs?

> It has only been tested with version .0.49.0 of scss_lint so we therefore suggest you run the following in your terminal: gem install scss_lint -v 0.49.0

Have you installed scss_lint in your system ruby? Or are you using rbenv and installed it in a projects ruby version? Installing it in the system ruby solved the problem for me back then.

**Comment by AndrewRayCode** [comment]
Looks like that did it :) I think the readme should be more strongly worded for people with poor reading comprehension like me

**Comment by adamwalzer** [comment]
Hey Andrew, could you suggest a better way to word the README? Thanks!

**Comment by adamwalzer** [comment]
Alright I'm going to close this issue, but if anyone has better ways of wording the README let me know. Thanks

**Comment by swsalim** [comment]
I'm having similar issue with what @AndrewRayCode reported initially.

Current Installed Versions:
scss-lint 0.49.0
vs code 1.10.1

I've downgraded the scss-link version to 0.49.0 and pointed "scssLint.configDir" to my home dir where my configuration is.

I've also tried to set "scssLint.showHighlights": true".

Still I'm not seeing any highlights or any error reported on the 'Problems' window and scss-lint reported 0 error and 0 warning on the footer.

The same file would return numerous error on Atom.

Any help on what I missed out?

Thanks.

**Comment by swsalim** [comment]
Updated! Managed to sort it out.

Initially installed the latest version of scss_lint before installing version 0.49.0. Apparently gem keeps both version and VS couldn't find which one to use, probably.

I removed the latest version and keep version 0.49.0 and it works.

**Comment by adamwalzer** [comment]
@swsalim thank you for pointing out and solving your issue. I'll look into trying to report issues like that better to the user.
